source/Xunit.Gherkin.Quick/Feature.cs

```diff
@@ -45,13 +45,27 @@ internal void Scenario(string scenarioName)
                 object[] methodParamValues = null;
                 if (methodParams.Length > 0)
                 {
-                    var methodParamStringValues = stepRegexMatch.Groups.Cast<Group>().Skip(1).Select(g => g.Value).ToList();
+                    var methodParamStringValues = stepRegexMatch.Groups.Cast<Group>().Skip(1).Select(g => g.Value).ToList<object>();
 
-                    if (methodParamStringValues.Count < methodParams.Length)
+                    if (methodParamStringValues.Count < methodParams.Length && parsedStep.Argument == null)
```

Comment:
This will miss the case when table argument is present but besides that count is much less. e.g. method expects 10 arguments, but supplied was 5, including table.